lambda/lib/moderator.js

```diff
@@ -29,7 +29,7 @@ module.exports = class Moderator {
             .then(result => {
                 console.log('The rekognition result:', util.inspect(result, {depth: 5}));
                 if (result.ModerationLabels.length > 0) {
-                    callback(new ModerationThresholdExceeded(result));
```

Comment:
This was driving me nuts in the logs. Doesn't have anything to do with https images though, lol.

Reply:
👍